pydantic_ai_slim/pydantic_ai/_parts_manager.py

```diff
@@ -67,6 +72,49 @@ def get_parts(self) -> list[ModelResponsePart]:
         """
         return [p for p in self._parts if not isinstance(p, ToolCallPartDelta)]
 
+    def _stop_tracking_vendor_id(self, vendor_part_id: VendorId | None) -> None:
```

Comment:
Let's move private methods to the very bottom so the class "gets to the point" quicker for someone (AI?) reading from top to bottom